source/state_representation/src/space/cartesian/CartesianPose.cpp

```diff
@@ -123,6 +135,25 @@ Eigen::VectorXd CartesianPose::data() const {
   return this->get_pose();
 }
 
+CartesianPose CartesianPose::inverse() const {
```

Comment:
why not use the CartesianState::inverse() and then downcast the return back to a pose? it would prevent some code duplication, and I would say the few extra linear operations are not hugely costly. It's also how the other operators are working anyway